.github/pull_request_template.md

```diff
@@ -6,8 +6,8 @@ This PR fixes #<issue ID>.
 
 How to test:
 
-- [ ] l10n dependencies have been merged, if any.
-- [ ] I've added a unit test to test for potential regressions of this bug (or this is a front-end change, where we don't yet have unit test infrastructure).
+- [ ] l10n changes have been submitted to the l10n repository, if any.
```

Comment:
They don't have to be merged anymore - the translations in `pendingTranslations.ftl` (which are not in the submodule) will be shown, and they'll be replaced by whatever's in the l10n repo as soon as it gets merged there.